index.js

```diff
@@ -49,8 +49,7 @@ class Toolbar {
         let $toolbar;
 
         let $defaultPlugins = plugins.default.map((Plugin) => { // eslint-disable-line no-unused-vars
-            // Render each plugin with the bound click handler
-            return <Plugin onClick={::this.handlePluginClick} />;
+            return <Plugin onClick={this.handlePluginClick.bind(this)} />;
```

Comment:
This function actually takes an argument, so `() => this.handlePluginClick()` is a no-go :( (without an argument, that is).

This changes in the refactoring I'm doing right now though.